frontend/src/metabase/querying/components/FilterValuePicker/SearchValuePicker/SearchValuePicker.tsx

```diff
@@ -63,12 +67,15 @@ export function SearchValuePicker({
       value={selectedValues}
       searchValue={searchValue}
       placeholder={placeholder}
-      nothingFound={isSearched ? nothingFound : null}
+      shouldCreate={shouldCreate}
+      getCreateLabel={searchValue => t`New ${searchValue}`}
```

Comment:
cc: @cdeweyx @mazameli 

1. "New" suggests that something will be created. I'd skip the word and just use the value itself.
2. It's not ideal when this value is last in the options list because it can't be seen when there are other results:
![image](https://github.com/metabase/metabase/assets/6830683/89f2be05-6e23-41a3-a17a-a2a38dcd15bd)

Reply:
3. Ideally the "new" value should not be displayed when an exact match is found:

![image](https://github.com/metabase/metabase/assets/6830683/da977319-b175-408b-9fde-393076e025b0)